jerry-port/common/jerry-port-io.c

```diff
@@ -55,7 +55,18 @@ jerry_port_print_buffer (const jerry_char_t *buffer_p, /**< string buffer */
 /**
  * Read a line from standard input as a zero-terminated string.
  *
- * @param out_size_p: length of the string
+ * The implementation should allocate storage necessary for the string. The result string should include the ending line
+ * terminator character(s) and should be zero terminated.
+ *
+ * An implementation may return NULL to signal that the end of input is reached, or an error occured.
+ *
+ * When a non-NULL value is returned, the caller will pass the returned value to `jerry_port_line_free` when the line is
+ * no longer needed. This can be used to finalize dynamically allocated buffers if necessary.
+ *
+ * This port function is never called from jerry-core directly, it is only used by some jerry-ext components that
+ * require user input.
+ *
+ * @param out_size_p: size of the input string in bytes, excluding terminating zero byte
```

Comment:
Yeah I was not really sure about it either and after some more digging it turns out doxygen will complain if even a single character is mismatched between the header and the .c file, for these files at least, I'm guessing because of a bug in doxygen.

For now I reverted the copy-paste and filtered out this warning.